soda/core/soda/soda_cloud/soda_cloud.py

```diff
@@ -179,6 +179,10 @@ def get_historic_data(self, historic_descriptor: HistoricDescriptor):
 
         return {"measurements": measurements, "check_results": check_results}
 
+    def get_check_attributes_schema(self) -> dict:
+        # TODO: implement.
```

Comment:
I think the attributes schema should be returned by the orgsettings call (which also contains failed rows enable/disable setting)

Reply:
removed